### PR TITLE
doc: Update command description for lxc import image

### PR DESCRIPTION
**Pull request by kadinsayani**
Fixes https://github.com/canonical/lxd/issues/13936.

https://documentation.ubuntu.com/lxd/en/latest/reference/manpages/lxc/image/import/#lxc-image-import-md will also be updated.

### CODE REVIEW

**Review by simondeziel — COMMENTED** [review]
The text addition is good IMHO but since it's in a `i18n` block, you likely need a follow-up commit to update the `.po` files (`make i18n`).

**Review by simondeziel — APPROVED** [review]

**Comment by tomponline** [comment]
spell checker failing

**Comment by kadinsayani** [comment]
> spell checker failing

It seems to be passing after a re-run.

File: lxc/image.go [hunk 1]
```diff
@@ -697,7 +697,9 @@ func (c *cmdImageImport) command() *cobra.Command {
 	cmd.Long = cli.FormatSection(i18n.G("Description"), i18n.G(
 		`Import image into the image store
 
-Directory import is only available on Linux and must be performed as root.`))
+Directory import is only available on Linux and must be performed as root.
+
+Descriptive properties can be set by providing key=value pairs. Example: os=Ubuntu release=jammy variant=cloud.`))
```
Comment (tomponline) [inline lxc/image.go:702]: please use "noble" here as that is current LTS

**Review by tomponline — APPROVED** [review]

**Pull request merged**